config/config.go

```diff
@@ -242,6 +243,18 @@ type RPCConfig struct {
 	// TCP or UNIX socket address for the RPC server to listen on
 	ListenAddress string `mapstructure:"laddr"`
 
+	// A list of origins a cross-domain request can be executed from.
+	// If the special '*' value is present in the list, all origins will be allowed.
+	// An origin may contain a wildcard (*) to replace 0 or more characters (i.e.: http://*.domain.com).
+	// Only one wildcard can be used per origin.
+	AllowedOrigins []string `mapstructure:"allowed_origin"`
```

Comment:
Should we prefix these options with `CORS`?

Reply:
Both in go and in config?

Reply:
Yes. They only make sense for cors, right?